recipe/meta.yaml

```
@@ -86,6 +93,10 @@ outputs:
   - name: flang
     script: install_flang.sh  # [unix]
     script: install_flang.bat  # [win]
+    build:
+      track_features:
+        - emscripten
+      string: {{ variant }}_h{{ PKG_HASH }}_{{ build_number }}  # [variant and variant.startswith("emscripten_")]
```

Since we don't have an explicit `build_number` variable here, you need to use
```suggestion
      string: {{ variant }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}  # [variant and variant.startswith("emscripten_")]
```

---

I had this variable and then I removed it :facepalm: 
It's restored now

---

It's not in the recipe now, and we have no need to introduce it. Even more so now when we don't need to distinguish variants. Please drop the variant, the build_num, and the build string changes.